src/test/java/org/opensearch/neuralsearch/processor/TextImageEmbeddingProcessorIT.java

```diff
@@ -61,6 +61,15 @@ public void testEmbeddingProcessor_whenIngestingDocumentWithNullMappingValue_the
         expectThrows(ResponseException.class, () -> ingestDocument(INDEX_NAME, INGEST_DOCUMENT.replace("\"passage_text_value\"", "null")));
     }
 
+    public void testEmbeddingProcessor_whenIngestingDocumentWithEmptyMappingValue_thenThrowException() throws Exception {
```

Comment:
Is it super necessary to add this validation in the form of integ test? Can't it be just a unit test? Doing that will save some build resource.

Reply:
+1, this can be validated via unit test, integ test with model is a bit expensive to run